README.md

```diff
@@ -14,19 +15,78 @@ Takeaway Challenge
+
+My Implementation
 -------
+I broke the user stories down into several interacting classes.
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+ - **Restaurant**: the main object that holds all others together.
+ - **OrderHistory**: a list of orders with the capability to generate new orders.
+ - **Order**: a list of food, and associated methods.
+ - **Good**: a parent class to **Food** and **Drink**, done to make the classes as universally applicable as possible for the projects longevity.
 
-Task
------
+All unit testing has been completed using mocking to ensure that the code would be easy to change, as well as to test Twilio without making a real API call. I've then implemented a demonstration of the classes working together in the './demo.rb' file.
+
+The idea was to make everything as modular and customisable as possible, to ensure that you could use this code as a jumping off point for a platform hosting multiple restaurants and serving multiple customers.
+
+Installing the app
+-------
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
+ - Clone the Github repo
+ - Register for a free Twilio account
+ - In the parent directory, create a file called .env
+ - Add to this file your Twilio API key, token, to and from numbers as outlined here:
+         
+       export TWILIO_ACCOUNT_SID='XXXXXXXXXXXXXXX'
+       export TWILIO_AUTH_TOKEN='XXXXXXXXXXXXXXX'
+       export TWILIO_TO=+44XXXXXXXXX
+       export TWILIO_FROM=+44XXXXXXXXX
+       
+ Example Usage
+-------
+
+    ❯ ruby ./demo.rb
+    Enter:
+    1 to see the menu
+    2 to add to your order
+    'confirm' to place your order
+    'quit' to cancel
+    > 1
+
+    Menu
+    -----------------------
+    Item #1: Garlic Bread - £4 (Starter)
+    Item #2: Olives - £2 (Starter)
+    Item #3: Halloumi Fries - £4 (Starter)
+    Item #4: Margherita Pizza - £8 (Main)
+    Item #5: Caesar Salad - £8 (Main)
+    Item #6: Ice Cream Sundae - £6 (Dessert)
+    Item #7: Beer - £4 (Drink)
+    Item #8: Cola - £3 (Drink)
+
+    Current order:
+    Your basket is empty
+    
+    Enter:
+    1 to see the menu
+    2 to add to your order
+    'confirm' to place your order
+    'quit' to cancel
+    > 2
+    Input the Item Number:
+    > 1
+    Enter the quantity you'd like:
+    > 5
+
+    Current order:
+    Item #1: Garlic Bread - £4 (Starter)
+    Quantity: 5
+    Total: £20
+
+Assignment Brief
+-------
 * Write a Takeaway program with the following user stories:
```

Comment:
Good example usage, as it guides the potential user through an example run showing all essential commands